Dockerfile.mysql

```diff
@@ -0,0 +1,7 @@
+FROM mysql:latest
+
+ENV MYSQL_DATABASE=app_db
```

Comment:
The `MYSQL_DATABASE` environment variable is set, but you should also set the `MYSQL_ROOT_PASSWORD` environment variable to ensure the database is secure. Consider using environment variables to avoid hardcoding sensitive information.